test_runner/batch_pg_regress/test_isolation.py

```diff
@@ -10,7 +10,9 @@ def test_isolation(zenith_simple_env: ZenithEnv, test_output_dir, pg_bin, capsys
     env.zenith_cli.create_branch("test_isolation", "empty")
     # Connect to postgres and create a database called "regression".
     # isolation tests use prepared transactions, so enable them
-    pg = env.postgres.create_start('test_isolation', config_lines=['max_prepared_transactions=100'])
+    pg = env.postgres.create_start(
+        'test_isolation',
+        config_lines=['max_prepared_transactions=100', 'max_replication_write_lag=15MB'])
```

Comment:
Should we apply this smaller lag limit in all the tests? It seems pretty accidental that we hit the problem with the isolation tests earlier, and not some other test.